crypto/rand/drbg_lib.c

```diff
@@ -194,7 +267,6 @@ int RAND_DRBG_uninstantiate(RAND_DRBG *drbg)
 {
     int ret = ctr_uninstantiate(drbg);
 
-    OPENSSL_cleanse(&drbg->ctr, sizeof(drbg->ctr));
```

Comment:
Is there a reason for this?

Reply:
It was moved into ctr_uninstantiate(), where it replaces a memset. To understand the change, its best to have a view at changeset https://github.com/openssl/openssl/pull/4402/commits/aec27cfcc5077171cc09344fe6d4aa6eab7479be